models/sipnet/R/model2netcdf.SIPNET.R

```diff
@@ -196,7 +196,7 @@ model2netcdf.SIPNET <- function(outdir, sitelat, sitelon, start_date, end_date,
      output[["Qle"]] <- (sub.sipnet.output$evapotranspiration * 10 * PEcAn.data.atmosphere::get.lv()) / timestep.s  # Qle W/m2
     }
     output[["Transp"]] <- (sub.sipnet.output$fluxestranspiration * 10) / timestep.s  # Transpiration kgW/m2/s
-    output[["SoilMoist"]] <- (sub.sipnet.output$soilWater * 10)  # Soil moisture kgW/m2
+    output[["SoilMoist"]] <- sub.sipnet.output$soilWater   # Soil moisture cm
```

Comment:
Violates PEcAn output standard. Changing the standard requires changing every coupled model

Reply:
Fixed.